From 6fe7addeb70cf5b5865fe2a5d81380d993924b45 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Tue, 7 Aug 2018 08:57:43 +0300
Subject: [PATCH 42/53] ARM: dts: imx7d: cl-som-imx7: remove TI shared
 transport driver

Texas Instrument recondition for the WL18xx part is to remove the shared
transport driver, and use just 'hci_uart' driver from the kernel:
https://e2e.ti.com/support/wireless_connectivity/
wilink_wifi_bluetooth/f/307/p/602155/2219156#2219156

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-cl-som-imx7.dts | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
index 3f41279..eca9a5d 100644
--- a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
@@ -85,19 +85,8 @@
 		startup-delay-us = <70000>;
 		enable-active-high;
 	};
-	/* Regulators - end */
-
-	kim {
-		compatible = "kim";
-		nshutdown_gpio = <&pca9555 1 GPIO_ACTIVE_HIGH>;
-		dev_name = "/dev/ttymxc2";
-		flow_cntrl = <1>;
-		baud_rate = <3000000>;
-	};
 
-	btwilink {
-		compatible = "btwilink";
-	};
+	/* Regulators - end */
 
 	leds {
 		compatible = "gpio-leds";
-- 
2.7.4

